scripts/build/build/target.py

```diff
@@ -285,8 +285,64 @@ def HumanString(self):
 
         return result
 
+    def CompletionStrings(self, value: str) -> List[str]:
```

Comment:
could we get some unit tests in https://github.com/project-chip/connectedhomeip/blob/master/scripts/build/build/test_target.py ?

Reply:
Done